src/FlexBox/FlexBox.js

```diff
@@ -14,6 +15,13 @@ export const FlexBox = ({ className, children, ref, ...props }) => {
   // Levers
   // ---------------------------------------------------------------------------
 
+  // props.element
+  let Element = 'div';
+
+  if (props.element && gridElements.includes(props.element)) {
```

Comment:
Shouldn't this be flexElements instead of gridElements?

Reply:
Yeah failed copy pasta 🍝